DocumentDataAPI/DocumentDataAPI/Data/Repositories/Helpers/ISqlHelper.cs

```diff
@@ -19,6 +21,14 @@ public interface ISqlHelper
     /// <example>string sqlQuery = "insert into document_contents (documents_id, content) values " + GetBatchInsertParameters(models, out Dictionary params);</example>
     string GetBatchInsertParameters<T>(T[] models, out Dictionary<string, dynamic> parameterDictionary);
 
+    /// Builds a string of SQL parameters for document searches, which may involve filters on several different attributes.
+    /// This method should be used for each parameter, and the calling method should handle whether the filter should start with "where" or "and".
+    /// </summary>
```

Comment:
Der ser ud til, at den gerne vil have end tag op på samme linje, for some reason.
```suggestion
    /// Builds a string of SQL parameters for document searches, which may involve filters on several different attributes.
    /// This method should be used for each parameter, and the calling method should handle whether the filter should start with "where" or "and".</summary>
```

Reply:
bing bong done